tasks

```diff
@@ -11,5 +11,8 @@ Before anything can be started, the following items must be finished.
 
 
 # Step 2
+# sub task 1
+
+# sub task 2
```

Comment:
there is one more task: task 3, maybe you can add it as well?

Reply:
done.